inngest/_internal/client_lib/client.py

```diff
@@ -240,6 +241,7 @@ def create_function(
             concurrency: Concurrency config.
             debounce: Debouncing config.
             fn_id: Function ID. Changing this ID will make Inngest think this is a new function.
+            idempotency: A key expression which is used to prevent duplicate events from triggering a function.
```

Comment:
**suggestion**: I know these descriptions don't cover everything, but it may be worth to add the 24h caveat to this.
```suggestion
            idempotency: A key expression which is used to prevent duplicate events from triggering a function over 24 hours.
```